landingai/pipeline/frameset.py

```diff
@@ -413,15 +417,27 @@ def is_empty(self) -> bool:
         """
         return not self.frames  # True if the list is empty
 
-    def run_predict(self, predictor: Predictor) -> "FrameSet":
+    def run_predict(self, predictor: Predictor, pipelining: bool = False) -> "FrameSet":
```

Comment:
Can we rename this `pipeelining` parameter to `max_workers: int = 1` instead? 

This way, we control directly if we want parallelization or not, and how many at a time (if `max_workers=1`, we don´t use the threadpool).

Reply:
I like the suggestion....I don't like the name...(too technical and void of domain meaning).... still I don't have a better name.. :(